core/jreleaser-model-impl/src/test/java/org/jreleaser/model/JReleaserModelTest.java

```diff
@@ -29,6 +29,7 @@
  * @author Geroge Gastaldi
  * @since 1.2.0
  */
+// TODO: viqueen - any reason this test is disabled ?
```

Comment:
I believe `GithubReleaser` requires more setup and mocks to be used in this test.